packages/graph/hash_graph/lib/graph/src/store/postgres/query/entity.rs

```diff
@@ -32,6 +32,11 @@ impl PostgresQueryRecord for EntityProperties {
 impl Path for EntityQueryPath<'_> {
     fn relations(&self) -> Vec<Relation> {
         match self {
+            Self::LeftEntity(path) | Self::RightEntity(path)
+                if **path == EntityQueryPath::Uuid || **path == EntityQueryPath::OwnedById =>
+            {
+                vec![]
+            }
```

Comment:
It's possible to add more special cases here, like `UpdatedById`, do we want that?

Reply:
This function will be removed in one of my next PR so I'd keep it as it is for now 🙂